### PR TITLE
Acc change

### LINKED ISSUE #159: Connect the change user variables page on sahaayak to the backend

### PR DESCRIPTION
**Pull request by ritwizsinha**
Fixes #159

Changes: 
Added change usrname to the backend

Surge Deployment Link: https://pr-159-sahaayak.surge.sh

Screenshots for the change: [Add here screenshots showing the changes.]

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@ritwizsinha Please check the commits. Previous PR changes are present too.

**Comment by ritwizsinha** [comment]
Done but this won't work unless the pr in the server is accepted

File: src/components/Settings/Account.js [hunk 1]
```diff
@@ -22,7 +22,15 @@ class Account extends Component {
   onChange = e => {
     this.setState({ [e.target.name]: e.target.value });
   };
-
+  onSubmit = ()=>{
+    const name = this.state.username;
+    if(name !== this.props.authUser.name){
+      const {id,email,isVendor,isAdmin} = this.props.authUser;
+      console.log(id,email,isVendor,isAdmin,name);
```
Comment (harsh-jindal) [inline src/components/Settings/Account.js:29]: Remove console.log

File: src/components/Settings/Account.js [hunk 2]
```diff
@@ -22,7 +22,15 @@ class Account extends Component {
...
+      const updatedUser = {name,email,isVendor,isAdmin};
```
Comment (harsh-jindal) [inline src/components/Settings/Account.js:30]: Just have name in this. No need of email, isVendor and isAdmin.

Comment (ritwizsinha) [reply src/components/Settings/Account.js:30]: Cannot remove this

**Pull request closed without merging**